srcpkgs/imwheel/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'imwheel'
+pkgname=imwheel
+version=1.0.0
+revision=1
+wrksrc="${pkgname}-${version}pre12"
+build_style=gnu-configure
+hostmakedepends="pkg-config libXmu libXext libXtst automake libtool"
```

Comment:
Remove `libXmu`, `libXext` and `libXtst`. They won't be required in the host (hostmakedepends), but the target (makedepends) where they are dependencies of their `*-devel` packages.